aspnet/mvc/views/partial/sample/global.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "projects": [ "src", "test" ],
+  "sdk": {
+    "version": "1.0.0-preview1-002702"
+  }
+}
```

Comment:
I think you can remove this too. @danroth27 we need to come up with a list of files to prune.